### DISCUSSION TITLE
Proposing workstreams

### DISCUSSION THREAD

**Discussion by justaugustus** [inclusivenaming/org#12] [Ideas]
As we grow the Inclusive Naming Initiative, we want to ensure that members are aligned with areas that:

- they're actually interested in
- allow them to leverage their industry

The [leadership team](https://inclusivenaming.org/leadership/) is opening this discussion thread to formally propose workstreams!

As you'll come to notice, I tend to draw many ideas from work in the Kubernetes Community.
Here, I'll liken the proposed workstreams to the [various types of Special Interest Groups (SIGs)](https://git.k8s.io/community/governance.md#community-groups):

- Vertical/Domain-specific
- Horizontal/Cross-project

Our proposed workstreams are as follows:

## Domain-specific

Each of the domain-specific workstreams will be focused around building process within the named domain:

- Domain-specific frameworks for evaluation
- Building and understanding implementation paths
- Establishing governance (as required)

### Open Source

**Lead(s):** @justaugustus / @celestehorgan

**Members:**

- @evalsocket
- @cpanato
- @geekygirldawn
- @mheis63
- @qnetter
- @spotz
- @jeschurch
- @mheis63
- @markcmiller86

### Company

**Lead(s):** @andrepellet

**Members:**

- @qnetter
- @samuelknuth
- @jasonbrooks
- @joannalee333
- @JasnaMRB

### [Standards Developing Organizations (SDOs)](https://en.wikipedia.org/wiki/Standards_organization#Standards_developing_organizations_(SDOs))

**Lead(s):**

**Members:**

- @richsalz
- @joannalee333
- @jasonkeeler11

## Program Management

Program Management workstreams focus on cross-Initiative efforts.

### Language

**Purpose:** Researching language, reviewing/approving recommendations

**Lead(s):** @celestehorgan 

**Members:**

- @JasnaMRB
- @markcmiller86
- @aserebrenik
- @quaid
- @gkarthiks
- @mheis63
- @joannalee333
- @lex-joy
- @jeschurch
- @hellasteph

### Community & Tooling

**Purpose:** Welcoming/directing new contributors, managing new contributions

**Lead(s):**

**Members:**

- Carla
- @richsalz
- @quaid
- @hellasteph

### Marketing

**Purpose:** Spreading the word, advertising, and content production

**Work items:**

- Press
- Blogs
- Social
- Recruitment

**Lead(s):** @xmulligan / @pritianka 

**Members:**

- @qnetter
- @Oshratn
- @gkarthiks
- @chris-short
- @shaberrry
- @lmaffeo

### Administration (restricted membership)

**Purpose:** Identity management

**Work items:**

- Website
- DNS
- Mailing list moderation
- Tooling maintenance
- YouTube
- Workstream definitions/management

**Lead(s):** @justaugustus / @edwarnicke 

**Members:**

- 

cc: @celestehorgan @edwarnicke @pritianka @xmulligan

**Comment by justaugustus** [comment]
If you are interested in participating in or leading one of these workstreams, please reply on the workstream threads below 👇🏾

**Comment by justaugustus** [comment]
Interested in the `Open Source` workstream? Reply here.

**Reply by yindia** [reply]
I would like to help in `open source` workstream.

**Reply by cpanato** [reply]
Changing my interest to `Open Source`. I would like to help in this workstream

**Reply by geekygirldawn** [reply]
I'm interested in helping with the open source workstream!

**Reply by mheis63** [reply]
I'm interested in helping with this workstream

**Reply by qnetter** [reply]
I’d like to help here. .

**Reply by spotz** [reply]
I'm interested in helping with the Open Source workstream

**Reply by jeschurch** [reply]
Definitely interested in helping in open source

**Reply by joannalee333** [reply]
I would also like to participate in the open source workstream

**Comment by justaugustus** [comment]
Interested in the `Company` workstream? Reply here.

**Reply by qnetter** [reply]
I’m not sure I understand it well enough.

**Reply by andrepellet** [reply]
Hi Stephen, as discussed - I am happy to lead this effort and organize a follow up to better define this workstream.

**Reply by samuelknuth** [reply]
I am interesting in signing up for this one

**Reply by jasonbrooks** [reply]
I'm interested in this one

**Reply by joannalee333** [reply]
I am interested in this work stream

**Reply by jameslongbotham** [reply]
Feel free to add me for this one!

**Comment by justaugustus** [comment]
Interested in the `Standards Developing Organizations (SDOs)` workstream? Reply here.

**Reply by cpanato** [reply]
I would like to help maybe in this workstream

**Reply by justaugustus** [reply]
Hey @cpanato! I just clarified that `Standards` is actually `Standards Developing Organizations (SDOs)` --> https://en.wikipedia.org/wiki/Standards_organization#Standards_developing_organizations_(SDOs).

With that clarification, are you still interested in this workstream?
I won't make any assumptions about your interests, but you _may_ be more interested in working on the `Open Source` workstream on efforts like Kubernetes WG Naming?

**Reply by cpanato** [reply]
thanks so much for sending to me this, i thought that was another thing, this looks like over my knowledge and profile.
Agree with you. will comment in the `Open Source`

**Reply by richsalz** [reply]
I would like to work on the SDO stream, focusing on IETF.

**Reply by joannalee333** [reply]
I work with many SDOs and would like to participate in this workstream

**Reply by jasonkeeler11** [reply]
I would like to work on the SDO stream.

**Reply by joannalee333** [reply]
If this workstream still needs a leader, I am happy to volunteer for this position.  I'm a legal counselor to dozens of SDOs.  I recently helped a well-known electronics SDO develop an inclusive language policy.  I look forward to contributing to this workstream!

**Reply by mschnoor-intel** [reply]
Hi!  I'm joining this workstream and I volunteer to serve as co-lead with Joanna.

**Comment by justaugustus** [comment]
Interested in the `Language` workstream? Reply here.

**Reply by tarkhil** [reply]
I still don't understand your goal. Are you trying to restrict a language to provide no possibility to offend anyone?

**Reply by celestehorgan** [reply]
✋

**Reply by JasnaMRB** [reply]
I'm interested in this workstream (as well as `Company` -- will select one over the other as I learn more).

**Reply by markcmiller86** [reply]
I am interested in this workstream

**Reply by aserebrenik** [reply]
I am interested in this workstream as well.

**Reply by quaid** [reply]
+1 I'd like to work on the `Language` workstream

**Reply by gkarthiks** [reply]
I am interested in the `Language` workstream

**Reply by quaid** [reply]
> I still don't understand your goal. Are you trying to restrict a language to provide no possibility to offend anyone?

No. And this is not the place for this question.

**Reply by mheis63** [reply]
I am interested in this worksteam as well as the Open Source one

**Reply by joannalee333** [reply]
I would like to participate in this workstream

**Reply by lex-joy** [reply]
I'd love to join the `Language` workstream. I come from a literature/language background, so this is completely my jam.

**Reply by jeschurch** [reply]
I'm interested in language workstream as this is what we're doing now at my company.

**Reply by hellasteph** [reply]
I'm interested in this work stream as a secondary interest or equal to Intake.

**Reply by KianaMinkie** [reply]
I'd love to join this workstream too.

**Comment by justaugustus** [comment]
Interested in the `Community & Tooling` workstream? Reply here.

**Reply by richsalz** [reply]
This would be my second choice; I've been involved in ietf newcomer's stuff for several years, and give the tutorials, e.g.

**Reply by quaid** [reply]
I am interested in the `Intake` workstream, as second or possibly equal to `Language` workstream.

**Reply by hellasteph** [reply]
I'm interested in this work stream but would recommend it be relabeled as "Community" to encompass a member's experience with on boarding and learning. Thanks!

**Reply by justaugustus** [reply]
Spoke with some of the leads (@celestehorgan @pritianka @edwarnicke @xmulligan) today and we're going to update this workstream name to `Community & Tooling`.
(Slack channel renamed to `ws-community-and-tooling`.)

Lots of great discussion has already been happening (https://github.com/inclusivenaming/org/discussions/12#discussioncomment-274957, https://github.com/inclusivenaming/org/discussions/18, https://github.com/inclusivenaming/org/discussions/14) and it sounds like both @markcmiller86, @edwarnicke, and @smokris (just to name a few in addition those who have already mentioned) are interested in this effort.

With that reframing, who else is interested in the `Community & Tooling` workstream?

**Reply by quaid** [reply]
> I am interested in the `Intake` workstream, as second or possibly equal to `Language` workstream.

I'll maintain interest here with the new framing

**Comment by justaugustus** [comment]
Interested in the `Marketing` workstream? Reply here.

**Reply by xmulligan** [reply]
👋

**Reply by qnetter** [reply]
Yes please.

**Reply by Oshratn** [reply]
I'd be interested in joining this workstream.

**Reply by xmulligan** [reply]
Just want to kick off this conversation in the New Year 🎆 @qnetter @Oshratn are there specific things you want to work on for marketing or do you have ideas of what we should do?

**Reply by Oshratn** [reply]
I think there needs to be a lot of evangelizing around naming. It's always very difficult for people to adjust to changes such as these (as opposed to being quick to adopt new tech 😉 ).
The most obvious way for me to help is blogging.

**Reply by xmulligan** [reply]
Great! @celestehorgan is setting up the blog right now

**Reply by xmulligan** [reply]
@Oshratn https://github.com/inclusivenaming/website/pull/31/files here is the PR for blogging

**Reply by gkarthiks** [reply]
I am interested in `Marketing`

**Reply by chris-short** [reply]
I can definitely help here.

**Reply by xmulligan** [reply]
For everyone that wants to help with marketing, please join the #ws-marketing channel in slack so we can start organizing there

**Reply by shahed-al** [reply]
Would love to participate here!

**Reply by lmaffeo** [reply]
I'd love to join this workstream!

**Comment by markcmiller86** [comment]
I am mainly a coder and would be interested in tooling workstream when it spins up.

As part of that, I am thinking any *adopted* set of terms deemed non-inclusive as well as suggested replacements that this effort (or others) publishes and maintains would be good to have codified into a *standard* file format that can be slurrped into various tools and workflows (npm, json, python, etc.). AFAIKT, the dashboard tools so far referenced here from Red-Hat's efforts aren't pitched at that level.

**Reply by edwarnicke** [reply]
@markcmiller86 Any interest in doing CI usable static-analysis style checkers?

**Reply by markcmiller86** [reply]
Sure.

**Reply by edwarnicke** [reply]
Excellent!  I've mostly been hanging out in Go lately, and using golangci-lint ... and it looks like it would be quite doable to write a plugin for that to cover inclusivenaming issues as part of CI static analysis :)  What would be your preferred language to start with, and what is its static analysis culture like?

**Reply by markcmiller86** [reply]
Preferred language? Probably English 😆. Well, I wish I was one of those types that had a large number to choose from. I don't know Go. I can do perl, bash or python pretty well. I figure no on is interested in the first two. So, probably python is best. OTOH, I would not want to let my limited programming language knowedge to somehow wind up being a deciding factor in what is chosen.

**Reply by justaugustus** [reply]
See here: https://github.com/inclusivenaming/org/discussions/12#discussioncomment-328798

**Comment by andrepellet** [comment]
Initial Enterprise/Company Workstream meeting this Thursday Feb 11 at 2 EST  - please email andre_p_pellet@dell.com to be added to the invite.
@qnetter 
@samuelknuth 
@jasonbrooks 
@joannalee333 
@JasnaMRB

**Comment by andrepellet** [comment]
KianaMinkie – Can you send me your email address?

**Reply by KianaMinkie** [reply]
Sure. I'm using a personal address for this project: 

kianaminkie21@gmail.com

Thanks!

**Comment by JimStLeger** [comment]
Wrt the SDO workstream:
I’m talking to my colleague Matthew Schnoor today about him stepping up to lead here (e.g. recruiting him 😉)
Perhaps he can tag-team with Joanna.

Best,
Jim

**Reply by joannalee333** [reply]
Fantastic, Jim.  Thank you.  If Matthew is open to leading I would be delighted to collaborate with him.

Best,
Joanna

**Reply by kdannels** [reply]
Hi @JimStLeger  @mschnoor-intel @joannalee333 - I'd like to join this workstream :) Sorry I am a bit late in raising my hand. Let me know if I've missed anything.

**Reply by mschnoor-intel** [reply]
Hi, kdannels!  It isn't too late.  Please join us in Slack and we'll bring you up to speed.

**Comment by MonicaRush** [comment]
@celestehorgan Hi -- I intended to join the language workstream. I've filled out the Doodle form for scheduling and hope to attend the upcoming kickoff. Just letting you know since I don't see my name on the list (@MonicaRush). If there's anything else I need to do please LMK. Thank you, Celeste!

**Comment by justaugustus** [comment]
All --

I think we've solved our initial goals (identifying the workstreams and leads, doing some recruiting for each, and starting to kickoff meetings).

If you're on Slack, here are the channels...

Pillar / vertical workstreams:
- #ws-standards (Standards / SDOs)
- #ws-open-source (Open Source)
- #ws-org (Enterprise/Company)

Program / horizontal workstreams:
- #ws-language (Language)
- #ws-community-and-tooling (Community and Tooling)
- #ws-marketing (Marketing)

If you're not on Slack, you should join us!
Here's the invite link: https://communityinviter.com/apps/inclusive-naming/invite

I'm going to lock and close this discussion, but questions/comments/concerns can be directed to:
- workstream-specific Slack channels
- GitHub issues/discussions
- the public mailing list

Details on our communication platforms can be found here: https://inclusivenaming.org/get-involved/

Looking forward to all of the work we'll get to do together!
